db/data/20230214121925_backfill_21_to_22_disabilities_from_apply.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class Backfill21To22DisabilitiesFromApply < ActiveRecord::Migration[7.0]
+  def up
+    upload = Upload.find_by(id: 6)
```

Comment:
are there any uploads in other deployed envs? maybe making sure this runs only in prod would be good. not sure whether it matters too much though!

Reply:
`production` and `productiondata` are the same and there's no uploads at all in the `staging` environment so I we're good.